recipe/meta.yaml

```diff
@@ -18,27 +18,39 @@ requirements:
   host:
     - python
     - pip
+    - setuptools
+    - wheel
+    - poetry-core
   run:
     - python
+    - leb128
+    - pytz
+    - lz4
+    - python-zstd
+    - tzlocal
+    - ciso8601
+  run_constrained:
+    - clickhouse-cityhash
```

Comment:
Does this do anything? I thought `run_constrained` meant _if_ the package was installed then make sure it was installed at a compatible version.

Reply:
Right, so this does nothing since any version is a compatible version if something else were to have already installed `clickhouse-cityhash` in the environment where `asynch` is being installed.

Reply:
indeed. `run_constrained` only means something if you've got a version range for the dep

Reply:
There's no version specified upstream, so this should be removed, or put into `run` if we think the functionality it provides is important and it should be provided by our conda package